src/common/comp/knowhere_config.cc

```diff
@@ -135,12 +135,12 @@ KnowhereConfig::SetAioContextPool(size_t num_ctx) {
 
 void
 KnowhereConfig::SetBuildThreadPoolSize(size_t num_threads) {
-    knowhere::ThreadPool::InitGlobalBuildThreadPool(num_threads);
+    knowhere::ThreadPool::SetGlobalBuildThreadPoolSize(num_threads);
```

Comment:
you can replace:
InitGlobalBuildThreadPool() / InitGlobalSearchThreadPool()
with:
SetGlobalBuildThreadPool() / SetGlobalSearchThreadPool()